packages/barracuda/0.1.0/manifest.yml

```diff
@@ -0,0 +1,31 @@
+format_version: 1.0.0
+name: barracuda
+title: Barracuda Web Application Firewall
+version: 0.1.0
+description: Barracuda Web Application Firewall Integration
+categories: [ "security" ]
+release: experimental
+removable: true
+license: basic
+type: integration
+conditions:
+    kibana:
+        version: '>=7.9.0'
+    elasticsearch:
+        version: '>=7.9.0'
+config_templates:
+- name: waf
+  title: Barracuda Web Application Firewall
+  description: Collect Barracuda Web Application Firewall logs from syslog or a file.
+  inputs:
+      - type: udp
+        title: Collect logs from Barracuda Web Application Firewall via UDP
+        description: Collecting syslog from Barracuda Web Application Firewall via UDP
+      - type: tcp
+        title: Collect logs from Barracuda Web Application Firewall via TCP
+        description: Collecting syslog from Barracuda Web Application Firewall via TCP
+      - type: file
+        title: Collect logs from Barracuda Web Application Firewall via file
+        description: Collecting syslog from Barracuda Web Application Firewall via file.
+# No icon
+icon:
```

Comment:
I'm pretty sure we can easily find a barracuda svg to dump on here, is there a reason not to add an icon?

Reply:
Afaik we don't have (yet) clearance from Barracuda for the use of their logo in a commercial product. I see a lot of logos in other packages, @mtojek is someone from the ingest manager team handling this? /cc @jamiehynds

Reply:
Nobody. Currently used logos are imported from existing open-source repos of Elastic. In general in terms of the ingest manager, the point of contact is @ruflin .

Reply:
So, I pulled the logo from https://www.elastic.co/integrations for my WIP integration. Even though I once went to law school, IANAL, so not entirely sure what additional consent we need from those other companies for inclusion of their logos here.

Reply:
According to @jamiehynds we're OK adding the logos. Will update the PR.